### PR TITLE
Fix element visibility check (see issue 869)

### PR DESCRIPTION
**Pull request by azanov**
Fixes the issue described in https://github.com/impresspages/ImpressPages/issues/869

### CODE REVIEW

File: Ip/Internal/Repository/assets/ipRepositoryAll.js [hunk 1]
```diff
@@ -111,7 +186,6 @@
                 // if term exists - loop all files
                 $files.each(function () {
                     var $file = $(this);
-                    var fileName = $file.data('fileData').fileName.toLowerCase();
```
Comment (jankus) [inline Ip/Internal/Repository/assets/ipRepositoryAll.js]: Why this is removed?

File: Ip/Internal/Repository/assets/ipRepositoryAll.js [hunk 2]
```diff
@@ -124,7 +198,6 @@
                 $this.find('.ipsBrowserSearch .fa-search').removeClass('fa-search').addClass('fa-times');
             } else {
                 // show all files if term doesn't exist
-                $files.removeClass('hidden');
```
Comment (jankus) [inline Ip/Internal/Repository/assets/ipRepositoryAll.js]: Why this is removed?

File: Ip/Internal/Repository/assets/ipRepositoryAll.js [hunk 3]
```diff
@@ -304,7 +377,6 @@
             }
 
             // filename
-            $file.find('span').text(data.fileName);
```
Comment (jankus) [inline Ip/Internal/Repository/assets/ipRepositoryAll.js]: Why this is removed?

File: Ip/Internal/Repository/assets/ipRepositoryAll.js [hunk 4]
```diff
@@ -538,8 +610,8 @@
                             // do not include already deleted files in the request, otherwise
                             // we'll end up in an endless loop, telling the user that some files
                             // could not be deleted (because they have been deleted already)
-                            $.proxy(methods._executeDelete, context)(files.filter(function(x) {
-                                return !~response.deletedFiles.indexOf(x.fileName);
+                            $.proxy(methods._executeDelete, context)(files.filter(function (x) {
+                                return ! ~response.deletedFiles.indexOf(x.fileName);
```
Comment (jankus) [inline Ip/Internal/Repository/assets/ipRepositoryAll.js]: Are you sure this space is needed?

File: Ip/Internal/Repository/assets/ipRepositoryAll.js (hunk 3 above)
Comment (azanov) [reply Ip/Internal/Repository/assets/ipRepositoryAll.js]: I had some custom code here, removed too much by accident

File: Ip/Internal/Repository/assets/ipRepositoryAll.js (hunk 2 above)
Comment (azanov) [reply Ip/Internal/Repository/assets/ipRepositoryAll.js]: I had some custom code here, removed too much by accident

**Comment by azanov** [comment]
I had some custom code in this file, so had to remove it before pushing and removed too much. Is fixed now.

**Comment by azanov** [comment]
Any updates? Can you merge this pull request?

**Comment by azanov** [comment]
push

**Pull request merged**